BLSQ/iaso
# WC2-461: Stabilize relabeling of types impact on ODK/workflows

Explain what problem this PR is resolving

Related JIRA tickets : [WC2-461](https://bluesquare.atlassian.net/browse/WC2-461)

## Self proofreading checklist


## Changes

Added new field 'code' in entity type and fill it with name content concatenated with account_id in **snake_case format**. The field value will never change once an entity type is created. Also, the ETL script will rely to the code instead of name which can change over the time.

## How to test

Run missing migrations on existing database with entities features. This will include the new migration for adding new field **code** in entity type table and fill automatically the field for existing data. Open api link for [entity types](http://localhost:8081/api/entitytypes/).

Then, create a new entity type and open api link for created entity [entity type](http://localhost:8081/api/entitytypes). You should see on the created entity type a value for code.

## Print screen / video


![Entity-Type-List-–-Django-REST-framework](https://github.com/user-attachments/assets/54c00b30-1597-42ed-89f3-eb6178961a84)


[WC2-461]: https://bluesquare.atlassian.net/browse/WC2-461?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
